http/util.go

```diff
@@ -21,7 +22,88 @@ import (
 )
 
 var (
+	restrictedAPIs = []string{
+		"sys/activation-flags/secrets-sync/activate",
+		"sys/audit",
+		"sys/audit-hash",
+		"sys/config/cors",
+		"sys/config/group-policy-application",
+		"sys/config/reload",
+		"sys/config/state",
+		"sys/config/ui",
+		"sys/decode-token",
+		"sys/experiments",
+		"sys/generate-recovery-token",
```

Comment:
As above:

```suggestion
```

Reply:
Ah, I was mistaken, this isn't about DR recovery but recovery unseal key shares. Resolving.